src/useLockBodyScroll.ts

```diff
@@ -15,15 +15,33 @@ export function getClosestBody(el: Element | HTMLElement | HTMLIFrameElement | n
   return getClosestBody((el as HTMLElement).offsetParent!);
 }
 
+function preventDefault(rawEvent: TouchEvent): boolean {
```

Comment:
This function prevents the onTouchMove from performing any functionality. I could rename this for clarity, since the only time we use this functionality is on the onTouchMove event listener.

Reply:
Something to also consider for this PR is how we handle use cases in which the onToucnMove has been changed on the client-side. This function will override this. 

Is this a cause for concern?

Reply:
LGTM